README.md

```diff
@@ -215,16 +215,18 @@ class Country
   include ElasticRecord::Model
 
   has_es_children(
-    join_field: 'pick_a_name_for_the_join_field',
-    children: ::ElasticRecord::Model::Joining::JoinChild.new(klass: State)
+    children:   State,
+    join_field: 'pick_a_name_for_the_join_field'
   )
 end
 ```
 
 `has_es_children` accepts an optional `name` argument, with a sane default. In the above example, it would default to `country`. The name can later be used to construct `has_parent` queries.
 ElasticRecord will define a getter method with the same name as the value provided to `join_field` on both the parent and all children (and grandchildren).
 
-`::ElasticRecord::Model::Joining::JoinChild.new` optional arguments:
+The `children` argument expects a Class, Array of Classes, or, for complex nestings, an instance of `::ElasticRecord::Model::Joining::JoinChild.new`.
```

Comment:
```suggestion
The `children` argument expects a Class or, for complex nestings, an instance of `::ElasticRecord::Model::Joining::JoinChild.new`.  You can also pass an Array, each element of which is either a `Class` or a `::ElasticRecord::Model::Joining::JoinChild.new`.
```

Also consider updating the example on line 218.  I suggest an Array of 2 elements, 1 a Class and the other a JoinChild.